### ISSUE TITLE
problem mappingProperties

### ISSUE THREAD

**Issue by ArtisanTinkerer** [elasticquent/Elasticquent#32]
Hi, I have just started using this package and I am getting:

 Fatal error: App\Part and Elasticquent\ElasticquentTrait define the same property ($mappingProperties) in the composition of App\Part. However, the definition differs and is considered incompatible. Class was composed in /var/www/html/testProject/app/Part.php on line 27

```php
 protected $mappingProperties = array(
        'PART_NO' => [
            'type' => 'string',
            "analyzer" => "standard",
        ],
        'DESCRIPTION' => [
            'type' => 'string',
            "analyzer" => "standard",
        ]

    );
```

Am I doing something wrong?

Mick

**Comment by timgws** [comment]
I would recommend setting `$mappingProperties` using the `setMappingProperties()` function inside your `__construct()`.

**Comment by TJTorola** [comment]
I get the same issue, using 1.0.6, but not 1.0.5.

**Comment by phpMick** [comment]
I just uploaded the mapping using curl instead.

**Comment by gthayden** [comment]
I am having the same issue, is there currently a fix for this?

**Comment by yaofong** [comment]
I am facing the same problem. Still no solution for this?

**Comment by timgws** [comment]
@yaofong this should be fixed, what version of Elastiquent are you using?

**Comment by paras-malhotra** [comment]
I have suggested a fix here: #42
